html/src/main/scala/org/hyperscala/html/tag/Video.scala

```diff
@@ -10,7 +10,7 @@ import org.hyperscala.html.constraints._
  * NOTE: This file has been generated. Do not modify directly!
  * @author Matt Hicks <matt@outr.com>
  */
-class Video extends Container[Source] with BodyChild with HTMLTag {
+class Video extends Container[BodyChild] with BodyChild with HTMLTag {
```

Comment:
I've encountered an HTML file that contains a paragraph within a ``<video>`` tag:

```html
<video ...>
  <p>To view this video please enable JavaScript.</p>
</video>
```